### PR TITLE
added the events intro for ruby workshop

### PR DESCRIPTION
**Pull request by krismyth**

### CODE REVIEW

File: events/0301_rb.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>A Day In The Life : Developers | Girl Develop It Ann Arbor</title>
```
Comment (jewlofthelotus) [inline events/0301_rb.html]: @krismyth Let's update the title to `Intro to Ruby`

File: events/0301_rb.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                    <h3 class="event__title">Intro to Ruby<br /><small><span class="green">4 Session Workshop</span></small></h3>
+                    <p><small>@gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #IntroToRuby</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+        </div>
+
+        <footer>
+            <div class="copyright">
+                #DayInTheLife &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; @gdiannarbor &nbsp;
```
Comment (jewlofthelotus) [inline events/0301_rb.html]: @krismyth Let's replace this #DayInTheLife hashtag

**Comment by jewlofthelotus** [comment]
@krismyth can you rename the file to `0315_rb.html`  - the convention we've started to work with is `monthyear_name.html`

File: events/0301_rb.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Event Intro -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title">Intro to Ruby<br /><small><span class="green">4 Session Workshop</span></small></h3>
+                    <p><small>@gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #IntroToRuby</small></p>
```
Comment (jewlofthelotus) [inline events/0301_rb.html]: @krismyth let's add Andrew Sardone and his @andrewa2 twitter name to the beginning of the line - here and in the closing slide

File: events/0301_rb.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                    <h3 class="event__title"><small>Tuesday 3/24 @ Corner Brewery</small></h3>
+                  <p><small>6:30pm - 8:30pm &nbsp; | &nbsp; 720 Norris, Ypsilanti</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <!-- Community Events -->
+            <section>
+                <h3>Community Events</h3>
+                <ul class="box--small copy--xsmall left" style="width: 48%">
```
Comment (jewlofthelotus) [inline events/0301_rb.html]: @krismyth lastly, let's clear out any events that take place on workshop days - 3/4 and 3/9; make sure the events are ordered by date and then i'll message you a few more events to add via Slack

**Comment by jewlofthelotus** [comment]
@krismyth one more - can you add a link to this new page on the main `events.html` page

As per the Trello card: "Add link to new event to the top of the lists in `events.html` in the root directory. You may need to create a new list for the month."

**Comment by krismyth** [comment]
Ah, yes. Will do.

File: events.html [hunk 5]
```diff
@@ -44,6 +44,11 @@
 
                     <p>Looking for slides or resources for a past event? <em>You've come to the right place!</em></p>
 
+                    <h3>March 2015</h3>
+                    <ul>
+                        <li>2/23/15 - <a href="events/0301_rb.html">Intro to Ruby</a></li>
```
Comment (jewlofthelotus) [inline events.html]: @krismyth we should probably use the correct date, too :)

**Pull request merged**